src/typings.d.ts

```diff
@@ -0,0 +1,4 @@
+/// <reference path="../typings/url-join/url-join.d.ts"/>
+
+// For a Thenable reference in jupyter-js-services
+type Thenable<T> = Promise<T>
```

Comment:
@jasongrout You mentioned in a review comment that this would eventually not be necessary. Is it still necessary in JupyterLab 0.11.2?